src/api/telemetry/TelemetryMetadataManager.js

```diff
@@ -78,6 +76,30 @@ define([
         valueMetadata.source = valueMetadata.source || valueMetadata.key;
         valueMetadata.hints = valueMetadata.hints || {};
 
+        if (valueMetadata.hints.hasOwnProperty('x')) {
+            console.warn(
+                'DEPRECIATION WARNING: `x` hints should be replaced with ' +
+                '`domain` hints moving forward.  ' +
+                'https://github.com/nasa/openmct/issues/1546'
+            );
+            if (!valueMetadata.hints.hasOwnProperty('domain')) {
+                valueMetadata.hints.domain = valueMetadata.hints.x;
+            }
+            delete valueMetadata.hints.x;
+        }
+
+        if (valueMetadata.hints.hasOwnProperty('y')) {
+            console.warn(
+                'DEPRECIATION WARNING: `y` hints should be replaced with ' +
+                '`range` hints moving forward.  ' +
+                'https://github.com/nasa/openmct/issues/1546'
+            );
+            if (!valueMetadata.hints.hasOwnProperty('range')) {
+                valueMetadata.hints.range = valueMetadata.hints.y;
+            }
+            delete valueMetadata.hints.y;
```

Comment:
Hints are checked first via `hasOwnProperty`, so `delete` is necessary to prevent the hints being used.  

https://github.com/nasa/openmct/blob/0a3d51c6b549da8d03dafa91191efb9d62e03268/src/api/telemetry/TelemetryMetadataManager.js#L154-L166